metaflow/includefile.py

```diff
@@ -300,7 +301,13 @@ class IncludeFile(Parameter):
     """
 
     def __init__(
-        self, name, required=False, is_text=True, encoding=None, help=None, **kwargs
+        self,
+        name: str,
+        required: bool = False,
+        is_text: bool = True,
+        encoding: str = None,
+        help: str = None,
```

Comment:
I saw you used `Optional` in other instances of parameters which could be `None`. Should `Optional` be used here as well?

Reply:
Good point. Yes, it probably should. I'll fix it.